textmining.py

```diff
@@ -0,0 +1,71 @@
+# most common words
+# this code computes the most common words of the adventure_of_sherlock_holmes
+import pickle
+import requests
+import string
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+
+adventure_of_sherlock_holmes = requests.get('http://www.gutenberg.org/ebooks/1661.txt.utf-8').text
+# Analyzing "Adventure of Sherlock Holmes"
```

Comment:
Is this part strictly necessary for this file? Considering that you have a separate data fetching file, it doesn't seem like this line of code is necessary.